app/assets/stylesheets/local/utilities.scss

```diff
@@ -0,0 +1,49 @@
+// utility classes used as overrides, therefore valid use of !important
```

Comment:
Like we discussed earlier, I'll defer to @axemonkey.

For the record, the CSS here is straight from https://github.com/ministryofjustice/hwf-publicapp/blob/master/app/assets/stylesheets/local/utilities.scss